code/modules/client/preferences.dm

```diff
@@ -945,12 +951,32 @@ var/const/MAX_SAVE_SLOTS = 10
 				if("pred_boot_type")
 					var/new_predator_boot_type = input(user, "Choose your greaves type:\n(1-4)", "Greave Selection") as num|null
 					if(new_predator_boot_type) predator_boot_type = round(text2num(new_predator_boot_type))
+				if("pred_mask_mat")
+					var/list/options = list("ebony", "silver", "bronze")
+					var/new_pred_mask_mat = tgui_input_list(user, "Choose your mask material:", "Mask Material", options)
+					if(!new_pred_mask_mat)
+						return
```

Comment:
Ditto

Reply:
Set to something at the top of the code if that works.